src/java.compiler/share/classes/javax/lang/model/util/Elements.java

```diff
@@ -707,6 +709,17 @@ boolean overrides(ExecutableElement overrider, ExecutableElement overridden,
      */
     Name getName(CharSequence cs);
 
+    /**
+     * {@return {@code true} if the element has an empty name, {@code false} otherwise}
+     *
```

Comment:
Please include an implSpec tag for returning false by default.

For testing the default method, see the use of VaccuousElements in tests in files like

test/langtools/tools/javac/processing/model/util/elements/TestRecordPredicates.java